doc/wg/network/README.md

```diff
@@ -0,0 +1,48 @@
+Network Working Group (NWG)
+=================================
+
+- Working Group Charter
+- Adopted 7/28/2023
+
+## Goals
+
+The goals of the Network Tock Working Group (NWG) are to:
+
+- design a set of interfaces (traits) for networking
+- design the user land interfaces for networking
+- define how buffer management should be implemented
+- determine other kernel infrastructure useful for networking
+
+## Members
+
+- Alexandru Radovici (Chair), OxidOS Automotive
+- Branden Ghena, Northwestern University
+- Leon Schuermann, Princeton University
+- Tyler Potyondy, UCSD
+- Ioan-Cristian Cirstea, OxidOS Automotive
+
+## Membership and Communication
+
+The networking working group membership is a subset of the people who have
+commit (pull request merge) permissions on the Tock repository. It is
```

Comment:
Is this true in practice (I think the answer is now) and do we want this to be the case? I don't see any hard requirement for this.

Reply:
```suggestion
The networking working group membership is open to Tock developers interested
in the design of network interfaces. It is
```
What about this?

Reply:
I don‘t see this addressed in the current version.